02-alouette/alouette.rb

```diff
@@ -0,0 +1,21 @@
+class Alouette
```

Comment:
The main benefit to providing a class is to guide them into storing the song components in an array. I'm not sure how important that bit of scaffolding will be.

If we wanted still to provide it but to make the format consistent with the other weekend warrior projects, we could throw the array definition in the README, otherwise I'll just delete the file.